src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories/Dashboard/DashboardRepository.cs

```diff
@@ -126,7 +88,7 @@ public BNumberModel GetBNumberModel(string bNumber, string label)
                 BNumber = bNumber,
                 DisplayTitle = label,
                 EncoreRecordUrl = string.Format(ddsOptions.PersistentCatalogueRecord, shortB),
-                ItemPageUrl = string.Format(ddsOptions.PersistentPlayerUri.Replace("/player/", "/item/"), bNumber),
+                ItemPageUrl = string.Format(ddsOptions.PersistentPlayerUri.Replace("/player/", "/item/"), bNumber), // TODO - can the setting just reflect this?
```

Comment:
Yeah, this is very legacy stuff, I will update and push to this branch.

Reply:
It's not even right for wellcomecollection going forward, so needs to be revisited.